core/utils/samples.py

```diff
@@ -758,7 +758,7 @@ def search_samples(sample_name, lab_name, sample_state, s_date, user):
         )
         sample_objs = sample_objs.filter(pk__in=state_ids)
     if s_date != "":
-        sample_objs = sample_objs.filter(sequencing_date_exact=s_date)
+        sample_objs = sample_objs.filter(sequencing_date=s_date)
```

Comment:
Just curious, do you know why it's not working with exact?

Reply:
Hmmmm could be that its `__exact` instead of `_exact`

Reply:
Yes! That's it! I knew something was off